src/api/app/views/webui/package/_deps.html.haml

```diff
@@ -85,3 +85,16 @@
           %tr
             %td{ colspan: '2' }
               %em No recommends
+
+  .table-responsive.col-sm-12.col-md-6.mt-4
+    %h4 Files
+    %table.table.table-bordered.table-hover.table-sm
+      %thead
+        %tr
+          %th Filename
+      - fileinfo.elements('filelist') do |package_file|
+        %tr
+          %td.text-word-break-all= package_file
+      - unless fileinfo['filelist']
```

Comment:
In this case, I would not even display a table, but only the heading and this _No files_ text in italic. What do you think?